metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableMetricModule.java

```diff
@@ -49,6 +49,15 @@
 public final class BigtableMetricModule implements MetricModule, DynamicModuleId {
     private static final String BIGTABLE_CONFIGURE_PARAM = "bigtable.configure";
 
+    /* default number of Rows for each batch mutation */
+    public static final int DEFAULT_MUTATION_BATCH_SIZE = 1_000;
```

Comment:
shouldn't the default be what it was previously? `10000`

Reply:
and before it said "maximum number of cells supported" so is this now referring to rows instead of cells or was the wording just not great previously?

Reply:
Yes and yes! Fixing now.